src/components/external/TransactionDetails/context/details/context.tsx

```diff
@@ -48,7 +48,7 @@ export const TransactionDetailsProvider = memo(
         const primaryAction = useCallback(() => {
             if (primaryActionDisabled) return;
             setActiveView(ActiveView.REFUND);
-            userEvents.addEvent('Switched to refund view', {
+            userEvents?.addEvent?.('Switched to refund view', {
```

Comment:
If I understand correctly `userEvents` is not optional, so the `?` is unnecessary.